src/labels/CustomLabels.labels

```diff
@@ -2127,6 +2135,38 @@
         <shortDescription>Plural Picklist substitute for Yearly</shortDescription>
         <value>Years</value>
     </labels>
+    <labels>
+        <fullName>RD2_EntryFormSaveCommitmentMessage</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Spinner text when Elevate Commitment is being created</shortDescription>
+        <value>Creating Elevate commitment... This might take some time, please wait.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_EntryFormSaveCommitmentFailedMessage</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Message displayed as prefix to errors returned from Elevate API</shortDescription>
+        <value>However, a Commitment in Elevate could not be created due to following error(s):</value>
```

Comment:
@RadSFDeveloper - What is the text that precedes line 2152 here? Trying to decide if it should start with "However" or not.

Reply:
The message before this one will be displayed in the same toast message. For example, the Recurring Donation is created but the Commitment in Elevate could not be created. We need to let user know details. Not keen on "However" but did not know how to connect these error messages indicating partial success. I tried to format the messages, but did not succeed but every message is displayed on the new line.

<img width="1423" alt="Screen Shot 2020-09-21 at 11 22 12 AM" src="https://user-images.githubusercontent.com/28743286/93805558-c5f94700-fbfc-11ea-97fa-5fd4de7ec0f3.png">